lib/cartopy/examples/barbs.py

```diff
@@ -6,6 +6,7 @@
 
 
 def main():
+    fig = plt.figure(figsize=(10, 5))
```

Comment:
FYI, I decided to add an explicit `figure` call in all the examples now, in order to more consistently use the object-oriented interface in Matplotlib.